src/game/AI/BaseAI/UnitAI.h

```diff
@@ -68,6 +68,7 @@ enum CastFlags
     CAST_MAIN_SPELL             = 0x100,                    // Marks main spell
     CAST_PLAYER_ONLY            = 0x200,                    // Selects only player targets - substitution for EAI not having more params
     CAST_DISTANCE_YOURSELF      = 0x400,                    // If spell with this cast flag hits main aggro target, caster distances himself - EAI only
+    CAST_TARGET_CASTING         = 0x800,                    // Selects only player targets that are casting
```

Comment:
Add  - EAI only comment here :D Only problem I have with it.

Reply:
Done :D